collections/_article/release-candidate-godot-4-1-4-and-4-2-2-rc-3.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Release candidates: Godot 4.1.4 RC 2 & 4.2.2 RC 3"
+excerpt: "Last round of release candidates for the upcoming maintenance releases."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/release-candidate-godot-4-1-4-and-4-2-2-rc-3.webp
+image_caption_title: "Sunken Shadows"
+image_caption_description: "A game by Sunrise Glitch Games"
+date: 2024-04-11 10:00:00
+---
+
+Here's the final round of release candidates for Godot 4.1.4 and 4.2.2, which I aim to release as stable in a few days.
+
+The previous [RC 2 builds](/article/release-candidate-godot-4-1-4-and-4-2-2-rc-2/) were already fairly solid, but [GDC 2024](/article/gdc-2024-retrospective/) and Easter holidays delayed the stable release a bit. And since a few more important bug fixes were merged in the meantime, I decided to do a final batch of cherry-picks and thus a third release candidate to validate them.
+
+These builds should be exactly what the stable release would be, so please give them a try with your current projects (and maybe give 4.2.2 RC 3 a spin during this weekend's [Ludum Dare 55](https://ldjam.com/events/ludum-dare/55) ;)). I won't wait too long this time to release the stable builds, so I need any potential regression to be reported in coming days.
+
+Maintenance releases are expected to be safe for an upgrade, but we recommend to always make backups, or use a version control system such as Git, to preserve your projects in a case of corruption or data loss.
```

Comment:
```suggestion
Maintenance releases are expected to be safe for an upgrade, but we recommend to always make backups, or use a version control system such as Git, to preserve your projects in case of corruption or data loss.
```

Reply:
I guess I'll fix that in the last 18 release posts :D
```
$ rg "in a case of corruption" | wc -l
18
```

Reply:
Fixed.

Reply:
first time I notice it 💀